fusillade-api.yml

```diff
@@ -688,6 +688,38 @@ paths:
           description: Unexpected error
       tags:
         - group
+    put:
+      summary: Add or remove a group from user(s).
```

Comment:
Maybe change this to "Add or remove user(s) from a group"?

Reply:
that's fair.